src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -47,13 +47,43 @@ class VirtualMachine(
         }
     }
 
+    fun wrapCode(code: MutableList<String>): MutableList<MutableList<String>> {
+        val wrappedCode = mutableListOf<MutableList<String>>()
```

Comment:
Can you extract these to a different file. These do not really need to be in the executor